src/Microsoft.Azure.WebJobs.Host/Triggers/TriggeredFunctionBinding.cs

```diff
@@ -56,7 +58,7 @@ internal class TriggeredFunctionBinding<TTriggerValue> : ITriggeredFunctionBindi
             {
                 ITriggerData triggerData = await _triggerBinding.BindAsync(value, context);
                 triggerProvider = triggerData.ValueProvider;
-                bindingData = triggerData.BindingData;
+                bindingData = triggerData.BindingData.Concat(AppSettingsBinding.CreateBindingData()).ToDictionary(x => x.Key, x => x.Value);
```

Comment:
Good point..I think in that case the AppSettings should supplement not override values that may already exist. Thoughts?